crypto.go

```diff
@@ -604,18 +606,20 @@ func computeFinishedData(params CipherSuiteParams, baseKey []byte, input []byte)
 	return mac.Sum(nil)
 }
 
-type keySet struct {
-	cipher aeadFactory
-	key    []byte
-	iv     []byte
+type KeySet struct {
+	Cipher AeadFactory
+	Key    []byte
+	Iv     []byte
+	Pn     []byte
```

Comment:
What is this thing?  If it's some QUIC-specific thing, I'm OK having it here if it's necessary, but please include a comment explaining it.

Reply:
I think that this object should move to `map[[]byte][]byte` or `map[string][]byte` and have the caller pass in a list of the keys that it wants generated.

Reply:
I just implemented this change.  It's pretty nice.